backend/src/services/encrypt/encrypt.service.ts

```diff
@@ -0,0 +1,19 @@
+import bcrypt from 'bcrypt';
+
+class Encrypt {
+  async createHash(data: string, salt: string): Promise<string> {
+    const hashedPassword = await bcrypt.hash(data, salt);
+    return hashedPassword;
```

Comment:
```suggestion
    return bcrypt.hash(data, salt);
```
It's not necessary to declare a variable only to immediately return it.